api/database/migrations/2022_08_09_164050_add_is_remote_column_pools_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIsRemoteColumnPoolsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pools', function (Blueprint $table) {
+            $table->boolean('is_remote')->default(false)->after('advertisement_location');
```

Comment:
I think this should be nullable?  Or should it be not-nullable in the schema?

Reply:
I'm actually not sure. My reasoning is that the position has to be remote or not and there is no real in between (null). Unless we treat null as not remote as well.

Reply:
Graphql tends to prefer nullable fields and therefore most of our database columns are also nullable.  It also using the muation as a "patch" with only some fields and allows us to force users to answer yes/no questions without making an assumption.  I'd recommend making it nullable and then making it a required field on the form.

Reply:
Okay, I have made it nullable in [make is_remote nullable](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3644/commits/4f61b22330e569c44c580419bacd2d0e612975d8). I also updated the publish validation to require this field now to prevent publishing when this field has not been selected.

The form still has this field marked as optional though since I'm not certain of the outcome of the discussion we had regarding which fields should be required while editing the pools. However, since it is a radio button, it would be very difficult for someone to submit the form without setting true/false.